Include/internal/pycore_obmalloc.h

```diff
@@ -253,14 +253,14 @@ typedef uint8_t pymem_block;
 /* Pool for small blocks. */
 struct pool_header {
     union { pymem_block *_padding;
-            uint count; } ref;          /* number of allocated blocks    */
-    pymem_block *freeblock;             /* pool's free list head         */
-    struct pool_header *nextpool;       /* next pool of this size class  */
-    struct pool_header *prevpool;       /* previous pool       ""        */
-    uint arenaindex;                    /* index into arenas of base adr */
-    uint szidx;                         /* block size class index        */
-    uint nextoffset;                    /* bytes to virgin block         */
-    uint maxnextoffset;                 /* largest valid nextoffset      */
+            uint count; } ref;          /* number of allocated blocks           */
+    pymem_block *freeblock;             /* pool's free list head                */
+    struct pool_header *nextpool;       /* see Pool table below for details     */
```

Comment:
I was misunderstood. The comments in the members should remain the same. What would change is:

```
/* Pool for small blocks (see "Pool table" below for details). */
struct pool_header {
```

Reply:
I think you are doing it right, I have modified it.

Reply:
> But at this point, I'm really wondering whether it's worth the change. Having an internal docs for that would be probably better though you'll need to navigate between files instead of having it at the code level.

As far as I know, the only detailed description of this structure is in the comments. I think it is worth.